.gitignore

```diff
@@ -4,6 +4,7 @@
 __pycache__/
 
 Build/
+Install/
```

Comment:
What is copied into folder install?

Reply:
```suggestion
```

I suggest to remove this due I confirmed that `Install` folder is not generated during the build.